### ISSUE TITLE
faker.unsplash is not defined

### ISSUE THREAD

**Issue by cdgn-coding** [faker-js/faker#412]
### Describe the bug

faker.unsplash is not defined whereas the docs allows to use it ([reference](https://fakerjs.dev/api/unsplash.html))

### Reproduction

Minimal code to reproduce:

```typescript
import faker from '@faker-js/faker'
faker.unsplash.technology()
```

This code shows a typescript warning
> Property 'unsplash' does not exist on type 'Faker'. ts(2339)

When it's run it throws this error
> TypeError: Cannot read properties of undefined (reading 'technology')

### Additional Info

I found that unsplash is accesible via the image property

```typescript
import faker from '@faker-js/faker'
faker.image.unsplash.technology()
```

We should either update the documentation or add unsplash to the Faker top level methods. The first option feels more natural to me. I would like to contribute introducing the fix if needed.

**Comment by ST-DDT** [comment]
If you are willing to contribute, then I can assign this issue to you.
The relevant code is in scripts/api-docs.ts.

**Comment by Shinigami92** [comment]
@cdgn-coding While we are fixing the docs, just use https://stackblitz.com/edit/typescript-wsqtmx?file=index.ts in the meantime

```ts
import { faker } from '@faker-js/faker';

faker.image.unsplash.avatar();
```

**Comment by cdgn-coding** [comment]
I started to use the methods below ```faker.image.unsplash``` and it seems to work. Nevertheless, I found that these methods always return the same URL ```https://source.unsplash.com/category/{category}/640x480``` and Unsplash has the responsibility to give randomized images under the category.  That doesn't seem to work as intuitively expected.

This [example](https://stackblitz.com/edit/typescript-tkqyuv?file=index.ts) renders images using ```faker.image.unsplash.technology()```,  and it's rare to find any that is related to technology. Whereas [Unsplash](https://unsplash.com/s/photos/technology) has many more technology-related examples. Do you think we can also improve this behavior in some sort of way?

**Comment by ST-DDT** [comment]
> I started to use the methods below `faker.image.unsplash` and it seems to work. Nevertheless, I found that these methods always return the same URL `https://source.unsplash.com/category/{category}/640x480`, and Unsplash has the responsibility to give randomized images under the category. That doesn't seem to work as intuitively expected.
> 
> This [example](https://stackblitz.com/edit/typescript-tkqyuv?file=index.ts) renders images using `faker.image.unsplash.technology()`, and it's rare to find any that is related to technology. Whereas [Unsplash](https://unsplash.com/s/photos/technology) has many more technology-related examples. Do you think we can also improve this behavior?

Yes, but any kind of bugfix of existing code is scheduled for 6.1 or later.
(We first want to have a stable foundation before touching anything.)

**Comment by Shinigami92** [comment]
Due to following code works, I will close this issue.

```ts
import { faker } from "@faker-js/faker";

faker.image.unsplash.technology();
```

Please open new dedicated issues for anything that was unrelated to this issue.

**Comment by ST-DDT** [comment]
This is a bug in our documentation, that needs to be fixed.

**Comment by airtonix** [comment]
Unsplash is deprecating the above urls in two stages: 

- stage one: `source.unsplash.com` only gives out random images from a extremely limited set of photos
- stage two: `source.unsplash.com` will 404

**Comment by ST-DDT** [comment]
We deprecated the unsplash apis for removal already.
https://github.com/faker-js/faker/blob/2e6b136c61a7cfc799c72a86a38ca4a505d94e3c/src/modules/image/index.ts#L12-L15
